lib/src/components/NxPolicyViolationIndicator/NxPolicyViolationIndicator.scss

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-policy-violation-indicator {
+  align-items: center;
+  background-color: #fff;
```

Comment:
Zeplin shows them as transparent

Reply:
> Zeplin shows them as transparent

I missed that: https://github.com/sonatype/sonatype-react-shared-components/pull/229/commits/61198f4e3a2082ed38d352e79ed376ccab2a2d6c